mod-orders-storage/examples/piece_get.sample

```diff
@@ -8,5 +8,6 @@
   "poLineId": "d471d766-8dbb-4609-999a-02681dea6c22",
   "receivingStatus": "Expected",
   "supplement": true,
+  "receiptDate": "2018-10-11T00:00:00.000Z",
```

Comment:
with "expected" as status, i don't think the the receivedDate could be populated